ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__metal-x86.yaml

```diff
@@ -1,8 +1,17 @@
+base_images:
+  cerberus.prow:
+    name: cerberus
+    namespace: chaos
+    tag: cerberus-prow
 build_root:
   image_stream_tag:
     name: ci-tools-build-root
     namespace: ci
     tag: latest
+images:
+- dockerfile_path: cerberus/Dockerfile_prow
```

Comment:
I think the issue is here when its trying to find the dockerfile and path its unable to find it in the openshift-qe/ocp-qe-perfscale-ci repo. I think if you want to use the cerberus observer you might need to put this test in the [redhat-chaos/prow-scripts folder ](https://github.com/openshift/release/tree/master/ci-operator/config/redhat-chaos/prow-scripts)

Reply:
Would it be possible to move on without the cerberus observer for now @paigerube14 ?

Reply:
yes definitely! do not need. Would just need to take out this images section and the observer from the test section below

Reply:
Great! This way we can move forward with the PoC for now.
@smanda99 can you please adjust the PR as per @paigerube14 's suggestion?
Thanks!

Reply:
Yes, sure i will do it. Thanks @paigerube14.